source/Paysafe/DirectDebit/StandaloneCredits.php

```diff
@@ -17,74 +17,78 @@
 
-        protected static $fieldTypes = array(
-            'id' => 'string',
-            'merchantRefNum' => 'string',
-            'amount' => 'int',
-            'ach' => '\Paysafe\DirectDebit\ACH',
-            'eft' => '\Paysafe\DirectDebit\EFT',
-            'bacs' => '\Paysafe\DirectDebit\BACS',
-            'profile' => '\Paysafe\DirectDebit\Profile',
-            'filter' => '\Paysafe\DirectDebit\Filter',
-            'billingDetails' => '\Paysafe\DirectDebit\BillingDetails',
-            'shippingDetails' => '\Paysafe\DirectDebit\ShippingDetails',
-            'customerIp' => 'string',
-            'dupCheck' => 'bool',
-            'txnTime' => 'string',
-            'currencyCode' => 'string',
-            'error' => '\Paysafe\Error',
-            'status' => array(
-                'RECEIVED',
-                'PENDING',
-                'PROCESSING',
-                'COMPLETED',
-                'FAILED',
-                'CANCELLED'
-            ),
-            'links' => 'array:\Paysafe\Link'
-        );
+    protected static $fieldTypes = array(
+        'id' => 'string',
+        'merchantRefNum' => 'string',
+        'amount' => 'int',
+        'ach' => '\Paysafe\DirectDebit\ACH',
+        'eft' => '\Paysafe\DirectDebit\EFT',
+        'bacs' => '\Paysafe\DirectDebit\BACS',
+        'profile' => '\Paysafe\DirectDebit\Profile',
+        'filter' => '\Paysafe\DirectDebit\Filter',
+        'billingDetails' => '\Paysafe\DirectDebit\BillingDetails',
+        'shippingDetails' => '\Paysafe\DirectDebit\ShippingDetails',
+        'customerIp' => 'string',
+        'dupCheck' => 'bool',
+        'txnTime' => 'string',
+        'currencyCode' => 'string',
+        'error' => '\Paysafe\Error',
+        'status' => array(
+            'RECEIVED',
+            'PENDING',
+            'PROCESSING',
+            'COMPLETED',
+            'FAILED',
+            'CANCELLED'
+        ),
+        'links' => 'array:\Paysafe\Link',
+        'splitpay' => 'array:\Paysafe\DirectDebit\SplitPay',
+    );
 
-        /**
-         *
-         * @param type $linkName
-         * @return \Paysafe\HostedPayment\Link
-         * @throws PaysafeException
-         */
-        public function getLink( $linkName ) {
-            if (!empty($this->link)) {
-                foreach ($this->link as $link) {
-                    if ($link->rel == $linkName) {
-                        return $link;
-                    }
+    /**
+     * @param string $linkName
+     * @return \Paysafe\Link
+     * @throws \Paysafe\PaysafeException
+     */
+    public function getLink( $linkName ) {
+        if (!empty($this->links)) {
+            foreach ($this->links as $link) {
+                if ($link->rel == $linkName) {
+                    return $link;
                 }
             }
-            throw new PaysafeException("Link $linkName not found in purchase.");
         }
-
+        throw new PaysafeException("Link $linkName not found in stand alone credit.");
```

Comment:
Looks like our exception message was copy/pasted. I updated to reflect the correct context.